packages/components/psammead-episode-list/src/fixtures.jsx

```diff
@@ -195,15 +195,9 @@ export const renderEpisodes = ({
                     locale: episode.locale,
                   })}`}
                 </span>
+                {episode.episodeTitle && episode.date}
```

Comment:
it's in acceptance criteria for the date to be outside of the link so that screenreaders don't read it out when tabbing links

Reply:
I think it will be easier to make it aria-hidden then

Reply:
but it's not supposed to be hidden from screenreaders. It still needs to be read out when you press the down arrow keys